html/publications.html

```diff
@@ -0,0 +1,222 @@
+
+<body>
+<header>
+    <!--NavBar-->
+    <nav class="navbar navbar-expand-lg navbar-light bg-light border-bottom">
+        <div class="container">
+            <a class="navbar-brand font-weight-bold" href="../index.html"><img src="../img/LogoUN.png"
+                                                                               class="logo-brand">Code</a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse"
+                    data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
+                    aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                <ul class="navbar-nav ml-auto">
+
+                    <li class="nav-item">
+                        <a class="nav-link" href="../index.html">Inicio<span class="sr-only"></span></a>
+                    </li>
+
+                    <li class="nav-item">
+                        <a class="nav-link" href="documentation.html">Documentación</a>
+                    </li>
+
+                    <li class="nav-item">
+                        <a class="nav-link" href="repositories.html">Repositorios
+                        </a>
+                    </li>
+                    <li class="nav-item active">
+                        <a class="nav-link" href="publications.html">Publicaciones
+                        </a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="contributors.html">Contribuidores</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="https://uncode.unal.edu.co/courselist">
+                            Cursos
+                            <svg width="1em" height="1em" viewBox="0 0 16 16" class="bi bi-arrow-up-right"
+                                 fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+                                <path fill-rule="evenodd"
+                                      d="M14 2.5a.5.5 0 0 0-.5-.5h-6a.5.5 0 0 0 0 1h4.793L2.146 13.146a.5.5 0 0 0 .708.708L13 3.707V8.5a.5.5 0 0 0 1 0v-6z"/>
+                            </svg>
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+</header>
+<!--Content-->
+<section>
+    <div class="container ">
+        <div class="row title">
+            <div class="col">
+                <h1 class="border-bottom py-4">Publicaciones</h1>
+                <ul class="list-group list-group-flush link-dark text-content title-content bg-light ">
+                    <li class="list-group-item border-bottom bg-grey">
+                        <h2 class="">UNCODE: INTERACTIVE SYSTEM FOR LEARNING AND AUTOMATIC EVALUATION OF COMPUTER
+                            PROGRAMMING SKILLS</h2>
```

Comment:
```suggestion
                        <h2 class="">UNCode: Interactive system for learning and automatic evaluation of computer programming skills</h2>
```
Check first with @ferestrepoca